apps/admin/pages/news/index.tsx

```diff
@@ -126,6 +135,18 @@ const NewsPage = () => {
         ),
       }}
     >
+      <ButtonGroup overflowX="auto">
+        {keywords[locale].map(keyword => (
+          <Button
+            key={keyword}
+            colorScheme={'primary'}
+            onClick={() => setSearchTerm(keyword)}
```

Comment:
Adding this trick can improve UX I guess.

```suggestion
            onClick={() => setSearchTerm(keyword)}
            variant={searchTerm === keyword ? 'solid' : 'outline'}
```